Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -30,6 +30,9 @@
 #include "PDFDocumentLayout.h"
 #include "PDFPageCoverage.h"
 #include "PDFPluginBase.h"
+#if PLATFORM(IOS_FAMILY)
+#include <WebCore/AXCoreObject.h>
+#endif // PLATFORM(IOS_FAMILY)
```

Comment:
You can get away with a forward declaration, so maybe shift the include from `UnifiedPDFPlugin.h` to `UnifiedPDFPlugin.mm`.

Reply:
The forward declaration would look like `class AXCoreObject;` under the `namespace WebCore` section in this header.